test/rebar_upgrade_SUITE.erl

```diff
@@ -768,8 +770,12 @@ run(Config) ->
     NewRebarConf = rebar_test_utils:create_config(filename:dirname(ConfigPath),
                                                   [{deps, ?config(next_top_deps, Config)}]),
     {ok, NewRebarConfig} = file:consult(NewRebarConf),
+    App1 = case App of
+        <<>> -> "--all";
+        _ -> App
+    end,
```

Comment:
This logic could be put in `normalize_unlock/1` and hide the implementation for all calls: https://github.com/erlang/rebar3/blob/fff5bb8a632e8a8127b9335b156964f3fe667e37/test/rebar_upgrade_SUITE.erl#L548-L553

Reply:
Moved the logic here, thanks!